src/plugin/ipc/file/fileconnlist.cpp

```diff
@@ -192,9 +192,17 @@ static vector<FileConnection *>shmAreaConn;
 
 void FileConnList::processReopen(int fd, const char *newPath)
 {
-  FileConnection *con = (FileConnection*) getConnection(fd);
-  if (con != NULL) {
-    con->updatePath(newPath);
+  Connection *con = getConnection(fd);
+  if (con == nullptr) {
+    return;
+  }
+
```

Comment:
Suggested comment (or something equivalent):
// User has already called freopen.  We are now updating the DMTCP bookkeeping to correspond.